Language fix - minor EN language errors (#191)
TYPO3-Documentation/TYPO3CMS-Tutorial-GettingStarted: Documentation/Troubleshooting/TYPO3.rst

Backend Administrator Password
------------------------------

> When the password for a backend user needs to be reset, log into the backend as a different user and use the :guilabel:`System > Backend Users` tool to reset
> the password. Note that only backend users with administrative rights can

Reviewer: ```suggestion
When the password for a backend user needs to be reset, log into the backend as a
different user and use the :guilabel:`System > Backend Users` tool to reset
the password. Note that only backend users with administrative rights can
```

access the `Backend Users` tool to make this change.

If an alternative administrator account is not available or it doesn't have the

> appropriate access, access the Install Tool directly

Reviewer: ```suggestion
appropriate access, open the Install Tool directly
```

Here _access, access_ sounds weird. ;-)

.. include:: ../Images/AutomaticScreenshots/InstallTool/InstallToolPassword.rst.txt

Once logged in to the Admin Tool go to :guilabel:`Maintenance > Create Administrative User`

> and select :guilabel:`Create Administrator`. In this dialog box you

Reviewer: ```suggestion
and select :guilabel:`Create Administrator`. Here you
```

During troubleshooting, in the :guilabel:`"Settings > Configuration Presets"`
section of the Install Tool, under "Debug settings", the "Debug" preset can be
changed to show errors in the frontend.

.. include:: ../Images/AutomaticScreenshots/DebugSettings/ConfigurationPresets.rst.txt

.. include:: ../Images/AutomaticScreenshots/DebugSettings/DebugSettings.rst.txt

> The following TypoScript setting can also be added into the root template for

Reviewer: ```suggestion
The following TypoScript setting can also be added to the root template for
```

.. important::

>    Once debugging has been completed, make sure you remove any debug Typoscript and

Reviewer: ```suggestion
   Once debugging has been completed, remove any debug TypoScript and
```

These files contain all :file:`ext\_tables.php` and
:file:`ext\_localconf.php` files of the installed extensions

> concatenated in the order that they are loaded. Therefore including one of

Reviewer: ```suggestion
concatenated in their loading order. Therefore including one of
```
